tests/input/constants-var.php

```diff
@@ -16,3 +18,12 @@ class Bar
 
     protected const PROPERTY_1 = '1', PROPERTY_2 = '2';
 }
+
+class Spacing
+{
+    public const FOO = 'bar';
+    public const BAR = 'bar';
+
+
+    public const BAZ = 'baz';
```

Comment:
What about constants with a comment?

Reply:
There is a sniff that removes those useless comments like `/** @var string */`

Otherwise, it behaves the same, min lines count is 0 and max lines count is 1.

Reply:
> those useless comments like

Nope, saying something like:

```php
/** State in which users are, when first registered to our website */
private const STATE_SEED = 1234;
```

How is spacing around it handled? Can it be added here, to prevent regressions?

Reply:
Sure, makes sense